onnxruntime/test/python/contrib_ops/onnx_contrib_ops_helper.py

```diff
@@ -0,0 +1,76 @@
+import onnx
```

Comment:
Also, please add a line or two saying what this script (and other newly added scripts) do.